opentelemetry-stackdriver/src/lib.rs

```diff
@@ -717,10 +730,13 @@ impl From<EvictedHashMap> for Attributes {
 
                 Some((key.to_owned(), v.into()))
             })
+            .take(32)
             .collect();
+
         Attributes {
             attribute_map,
-            dropped_attributes_count,
+            dropped_attributes_count: dropped_attributes_count
+                + (num_resource_attributes + num_attributes).saturating_sub(32) as i32,
```

Comment:
nit: can we make 32 a constant? Similar in L733

Reply:
Seems to have been done now.